source/server/main.js

```diff
@@ -366,13 +365,17 @@ function handleStartRound(webSocketConnection) {
   }
 
   // TODO: validate request is coming from host
-
-  gameInstance.gameState.byRound.push(createNewRound());
+  const newRound = createNewRound(gameInstance);
+  gameInstance.gameState.byRound.push(newRound); //creates world event in NewRound
 
   gameInstance.webSocketConnections.forEach((webSocketConnection) => {
     sendMessage(webSocketConnection, {
       action: S2C_ACTIONS.START_ROUND,
     });
+    sendMessage(webSocketConnection, {
```

Comment:
Maybe abstract this so that we aren't sending world events from two different places in the code